### DISCUSSION TITLE
do u support 'lock' function

### DISCUSSION THREAD

**Discussion by echolove38** [SVG-Edit/svgedit#715] [New feature request]
when user click the 'lock' button , the selected element couldn't be moved and resized, etc
I think this is a normal demand, and it's useful, hope svgedit support it

**Comment by jfhenon** [comment]
Hello, 
(I moved your question to a discussion).
Locking an element is not supported by the SVG standard so we are not going to support this. Obviously, any PR that would implement such proposal would be considered. I guess it could be an extension.
Thanks

**Reply by suguoyao** [reply]
Could you please provide a feasible solution? 😄
I can think of a solution that is to prevent layers from being clicked and selected.

**Reply by jfhenon** [reply]
Yes, this sounds like a possible solution indeed. However, I'm not able to invest into this but would be happy to review and merge proposals about such feature.

**Comment by henjoe** [comment]
Hi is there any progress related to this? This is highly usefule.

**Comment by w8w8w8** [comment]
<img width="1920" height="1030" alt="image" src="https://github.com/user-attachments/assets/c499d752-df80-4821-b568-d07ebf2837e5" />
My project is developed using Angular. I added a locking property, and then I check if it's locked when the user selects and drags an element with the mouse.

**Reply by henjoe** [reply]
Thanks, I think fuxa has this already.
